### ISSUE TITLE
Panel does not work with ipywidgets 8.0.0

### ISSUE THREAD

**Issue by govinda18** [holoviz/panel#4060]
#### ALL software version info
panel 0.14.0, ipywidgets 8.0.0

#### Description of expected behavior and the observed behavior
Panel does not render with ipywidgets 8.0.0, works fine with 7.7.x

#### Complete, minimal, self-contained example code that reproduces the issue

```python
import panel as pn
import ipywidgets as w
pn.extension()
pn.Column(w.HTML("Asd"))

```

#### Stack traceback and/or browser JavaScript console output
Uncaught (in promise) Error: Module @jupyter-widgets/controls, semver range ^2.0.0 is not registered as a widget module

#### Screenshots or screencasts of the bug in action
![image](https://user-images.githubusercontent.com/25079291/198202703-f93c6fbd-da9c-4249-aab4-039f7147c5f6.png)

**Comment by itsgifnotjiff** [comment]
Can confirm that although the widgets appear in the notebook any `servable` component is not showing.  I have tried on Linux and on Windows using both Panel `0.14` and `0.14.1` and `ipywidgets=8.0.2` through a Panel server, JupyterLab and JupyterHub-JupyterLab instance.

**Comment by philippjfr** [comment]
Weirdly I cannot reproduce:

<img width="661" alt="Screen Shot 2022-10-28 at 12 07 25" src="https://user-images.githubusercontent.com/1550771/198562403-fb448176-b8cf-47cc-a3a0-007a96ba5756.png">

**Comment by govinda18** [comment]
I tried to create an isolated reproducer using pure python venv `python -m venv panel-bug-venv`, activate it `source `panel-bug-venv/bin/activate`, install it as ipykernel `python -m ipykernel install --user --name=panelbugkernel`, install panel and ipywidgets (exact freeze requirements are below) and launch lab from there.

<details name="Requirements">
asttokens==2.0.8  

backcall==0.2.0
bleach==5.0.1
bokeh==2.4.3
certifi==2022.9.24
charset-normalizer==2.1.1
debugpy==1.6.3
decorator==5.1.1
entrypoints==0.4
executing==1.1.1
idna==3.4
importlib-metadata==5.0.0
ipykernel==6.16.2
ipython==8.5.0
ipywidgets==8.0.2
jedi==0.18.1
Jinja2==3.1.2
jupyter_client==7.4.4
jupyter_core==4.11.2
jupyterlab-widgets==3.0.3
Markdown==3.4.1
MarkupSafe==2.1.1
matplotlib-inline==0.1.6
nest-asyncio==1.5.6
numpy==1.23.4
packaging==21.3
panel==0.14.0
param==1.12.2
parso==0.8.3
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.2.0
prompt-toolkit==3.0.31
psutil==5.9.3
ptyprocess==0.7.0
pure-eval==0.2.2
pyct==0.4.8
Pygments==2.13.0
pyparsing==3.0.9
python-dateutil==2.8.2
pyviz-comms==2.2.1
PyYAML==6.0
pyzmq==24.0.1
requests==2.28.1
six==1.16.0
stack-data==0.5.1
tornado==6.2
tqdm==4.64.1
traitlets==5.5.0
typing_extensions==4.4.0
urllib3==1.26.12
wcwidth==0.2.5
webencodings==0.5.1
widgetsnbextension==4.0.3
zipp==3.10.0
</details>

I was able to consistently reproduce it. Let me know if this helps.

**Comment by itsgifnotjiff** [comment]
> Weirdly I cannot reproduce:
> 
> <img alt="Screen Shot 2022-10-28 at 12 07 25" width="661" src="https://user-images.githubusercontent.com/1550771/198562403-fb448176-b8cf-47cc-a3a0-007a96ba5756.png">

Are you able to Panel serve [the Panel ipywidgets examples](https://panel.holoviz.org/reference/panes/IPyWidget.html) or to preview them in any way except inline Mr. @philippjfr ? I have been able to at least view them in the notebook if I use `display( ipywidget_object `)  but any templating or serve attempt renders a blank page (not even the header portion without the components in `main` .

```python
from ipyleaflet import Map, VideoOverlay
import panel as pn
pn.extension() # tried 'ipywidgets' which makes it worse for me with a JS error even within the Notebook

m = Map(center=(25, -115), zoom=4)

video = VideoOverlay(
    url="https://www.mapbox.com/bites/00188/patricia_nasa.webm",
    bounds=((13, -130), (32, -100))
)

m.add_layer(video);

map_panel = pn.panel(m)

bootstrap = pn.template.BootstrapTemplate(title='ipyleaflet Demo')
bootstrap.main.append(pn.Row(pn.pane.Markdown("""# Title""")))
bootstrap.main.append(map_panel)

bootstrap.servable()
```

```
ipywidgets : 8.0.2
ipyleaflet : 8.0.2
Panel : 0.14.0
```

Although I also tried using Panel `0.14.1rc...` and the issue is consistent.
![image](https://user-images.githubusercontent.com/50274281/198587493-1e022c71-496c-4cad-9eb7-ceb3910c1655.png)

**Comment by philippjfr** [comment]
Nothing but inline in a notebook works for me currently because `ipywidgets_bokeh` has to be updated to support ipywidgets 8.0, so there's definitely work to do. Curious that notebook usage isn't working for you though, I'll follow your instructions on reproducing it when I get a chance.

**Comment by itsgifnotjiff** [comment]
> Nothing but inline in a notebook works for me currently because `ipywidgets_bokeh` has to be updated to support ipywidgets 8.0, so there's definitely work to do. Curious that notebook usage isn't working for you though, I'll follow your instructions on reproducing it when I get a chance.

Here is the JS error if that helps as well Mr. @philippjfr 
```
panel : 0.14.1rc2.post3+g1647fa3f
ipyleaflet : 0.17.2
ipywidgets : 8.0.2
```

```
[Open Browser Console for more detailed log - Double click to close this message]
Failed to load view class 'ErrorWidgetView' from module '@jupyter-widgets/base'
Error: No version of module @jupyter-widgets/base is registered
    at m.loadClass (https://arkham:8015/user/afsmrdv/static/lab/6187.a6425da91a9e3b166c3f.js?v=a6425da91a9e3b166c3f:1:4022)
    at m.loadViewClass (https://arkham:8015/user/afsmrdv/static/lab/4416.9d6d0a2f3f9ed5d7b141.js?v=9d6d0a2f3f9ed5d7b141:1:11555)
    at https://arkham:8015/user/afsmrdv/static/lab/4416.9d6d0a2f3f9ed5d7b141.js?v=9d6d0a2f3f9ed5d7b141:1:4289
    at async r.lazy_initialize (:1587:625)
    at async c (:437:152)
    at async t.build_views (:437:530)
    at async _.build_child_views (:655:1788)
    at async _.lazy_initialize (:655:658)
    at async c (:437:152)
    at async t.build_view (:437:257)
```

**Comment by philippjfr** [comment]
@itsgifnotjiff That seems like a misconfiguration of ipywidgets and Jupyter on your end and not an issue with Panel. I'd recommend creating a clean install of jupyter+ipywidgets+ipyleaflet. 

We've now also released ipywidgets_bokeh 1.3 and Panel 0.14.2 due to be released soon has another fix for ipywidgets that will resolve some further issues. Therefore I'll consider this as resolved.
